test/DynamoCoreWpfTests/CrashReportingTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using Dynamo.ViewModels;
+using NUnit.Framework;
+
+namespace Dynamo.Tests
+{
+    public class CrashReportingTests : DynamoViewModelUnitTest
+    {
+        public List<string> TargetWords = new List<string> { "New Issue", "DynamoDS" };
```

Comment:
private please if possible. It's low importance because this is a test file, but it's good to start conservatively.